### ISSUE TITLE
Documentation: "How to choose a model" ideas

### ISSUE THREAD

**Issue by dcorre** [karllark/dust_attenuation#29]
As the attenuation law is not universal, one should use a model allowing different UV slope, 2175 bump strength whenever the spectral coverage allows it. For this purpose, one of the most used recipe is the one of Noll+09.

- In particular cases, for instance the SED fitting of a galaxy known to be starburst and a lack of rest-frame UV data, the Calzetti law can be used.

- One should always avoid to use recipe derived from measurements of individual stars (i.e extinction curves) such as the average LMC, SMC or MW extinction laws. 

- The functional forms of FM90 or CCM89 are just analytical formulas and so even if they were initially applied to measurement of individual stars they could in principle also be used for galaxies. The FM recipe has a lot of parameters to fit though and setting the parameters to average values derived from measurement of individual stars is not recommended as the physical processes at stake are not the same. 

- The Charlot and Fall 2000 recipe is dependent on the star formation history

- Attenuation curves derived with radiative transfer models are described by physically motivated parameters related to the structure of the ISM: dust/star geometry, local distribution of dust, type of dust grain, optical depth.

**Comment by karllark** [comment]
Your comment about the FM90 and CCM89 forms gave me a thoughts.

I wonder if the FM90 (UV only) and CCM89 shapes can describe the expected shapes of attenuation curves as predicted from radiative transfer models.  Not all as CCM89 cannot reproduce anything w/o a 2175 A bump.  To these possible models, I would add G16 and P92.   I do not know the answer and it would be very interesting to know.  And would make a contribution to the field of dust attenuation in galaxies and a nice paper (or part of a paper).  If nothing else, this could be part of a paper describing the dust_attenuation package (maybe 1 paper for dust_attenuation and dust_extinction)?

**Comment by dcorre** [comment]
That's one of the reason why I'm interested in this package, to study which attenuation recipe is able to reproduce the attenuation curves computed with radiative transfer models. Here in Marseille, we have performed a similar analysis this with V. Buat for Noll09 and SD16. Paper is in preparation. It would be very interesting to extend this analysis to different recipes and radiative transfer models. This would be very nice to have a paper presenting both packages, addressing the differences between the two and showing which attenuation recipe is the most capable of reproducing attenuation curves from radiative transfer models.

**Comment by karllark** [comment]
Great that we agree that a paper or papers focused on the two packages.  Now to just find the time.  :-)

**Comment by dcorre** [comment]
Time is a concern for me at the moment, I have 2 and a half months left for writing my PhD manuscript...
Anyway, before starting the paper one need to add more content in dust_attenuation ;)
I will add the Noll09 law and SD16 radiative transfer models by mid-July. 

Regarding the ideas I wrote above, these are what I plan to add in the "how to choose" doc. I posted them to check with you whether there are relevant. If anyone else wants to start it, this is of course perfectly fine :)

**Comment by karllark** [comment]
Completely reasonable.  You should definitely focus on your PhD!  Good luck.
